

sound/core/control.c:1102: undefined reference to `unlock_kernel'
sound/built-in.o(.text+0x4de6):sound/core/control.c:1104: undefined reference to `lock_kernel'
sound/built-in.o(.text+0x4e3e):sound/core/control.c:1104: undefined reference to `lock_kernel'
sound/built-in.o(.text+0x4ea3):sound/core/control.c:1104: undefined reference to `lock_kernel'
sound/built-in.o(.text+0x4f7a):sound/core/control.c:1104: undefined reference to `lock_kernel'

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/sound/core/control.c |    1 +
 1 files changed, 1 insertion(+)

diff -puN sound/core/control.c~sound-control-build-fix sound/core/control.c
--- 25/sound/core/control.c~sound-control-build-fix	2004-08-06 20:21:12.576366648 -0700
+++ 25-akpm/sound/core/control.c	2004-08-06 20:21:12.580366040 -0700
@@ -26,6 +26,7 @@
 #include <linux/slab.h>
 #include <linux/vmalloc.h>
 #include <linux/time.h>
+#include <linux/smp_lock.h>
 #include <sound/core.h>
 #include <sound/minors.h>
 #include <sound/info.h>
_
